bedrock/urls.py

```diff
@@ -3,7 +3,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from django.conf import settings
-from django.conf.urls import handler404, include, url
+from django.urls import handler404, include, re_path
```

Comment:
Tests are failing because it appears that `handler404` is not available to be imported from `django.urls`

`ImportError: cannot import name 'handler404' from 'django.urls' (/venv/lib/python3.7/site-packages/django/urls/__init__.py)`